app/DoctrineMigrations/Version20170811124421.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20170811124421 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
```

Comment:
I don't know who started this tradition, but let's stop. Our tests run on sqlite...

Also: `addSql()` is a very blunt way to use the doctrine migrations API, there are calls like 'addColumn()' and 'addIndex()' that take care of a lot of the 'magic numbers' like 'UNIQ_8D93D6493AE5C753' and column comments used in the queries.